packages/scrum-client/src/components/Footer.js

```diff
@@ -1,19 +1,64 @@
-import React from 'react';
-import { Container, Row, Col } from 'react-bootstrap';
+/* eslint-disable no-underscore-dangle */
+/* eslint-disable no-console */
+import React, { useState, useEffect } from 'react';
+import { Row, Col, Container } from 'react-bootstrap';
+import { END_POINTS } from 'scrum-common';
+import ToastWrapper from './ToastWrapper/ToastWrapper';
+import { API_CONSTANTS } from '../constants';
 
-const Footer = () => (
-  <footer className="footer">
-    <Container>
-      <Row className="text-center">
-        <Col md={12} className="py-5">
-          <div className="mb-5 flex-center" />
-        </Col>
-        <Col md={12}>
-          <small className="d-block mb-3 text-muted">© 2020 Copyright: kinnetik-mx.com</small>
-        </Col>
-      </Row>
-    </Container>
-  </footer>
-);
+const TIMEOUT = 10000;
```

Comment:
I think const TIMEOUT = 10000; should not be part of footer , it should be in the wrapper of the toast. 
So all constants, logic related to it are within the same wrapper. 
What do you think ?

Reply:
Makes sense